Sources/Starknet/Crypto/FeeCalculation.swift

```diff
@@ -1,19 +1,44 @@
 import BigInt
 import Foundation
 
-/// Add overhead to estimated fee
+/// Convert estimated fee to max fee with added overhead
 ///
 /// Add overhead to estimated fee. Calculates multiplier as m = round((1 + ovehead) \* 100%).
 /// Then multiplies fee by m and does integer division by 100.
 ///
 /// - Parameters:
 ///  - fee: originally estimated fee
-///  - overhead: how big overhead should be added (as a fraction of fee) to the fee, defaults to 0.1
-/// - Returns: fee with added overhead
-public func estimatedFeeToMaxFee(_ fee: Felt, overhead: Double = 0.1) -> Felt {
-    let multiplier = BigUInt(Int((1.0 + overhead) * 100))
+///  - overhead: how big overhead should be added (as a fraction of fee) to the fee, defaults to 0.5
+/// - Returns: max fee with added overhead
+public func estimatedFeeToMaxFee(_ fee: Felt, overhead: Double = 0.5) -> Felt {
+    addOverhead(fee.value, overhead).toFeltClamped()
+}
```

Comment:
Should we remove this in favor of `StarknetFeeEstimate.toMaxFee`?

Reply:
I think so, yes